TitanRobotics2024/src/main/java/frc/robot/Subsystem/ModifiedEncoders.java

```diff
@@ -21,23 +23,27 @@ public ModifiedEncoders(int channelA, int channelB, String encodertype)
             case "E4TEncoder":
                 encoder = initializeE4T(channelA, channelB);
                 break;
-            case "QuadratureEncoder":
+            case "QuadratureEncoder": //relative encoder
                 encoder = initializeQuadrature(channelA, channelB);
                 break;
             default:
-                System.err.println("Error: encoders not activated");
+                System.err.println("Error: dual channel encoders not activated");
                 encoder = null;
         }
     }
 
-    public ModifiedEncoders(int portNumber, String encodertype)
+    public ModifiedEncoders(int portNumber, double positionOffset, String encodertype)
     {
         switch (encodertype)
         {
             case "CANSparkMaxEncoder":
                 break;
+            case "DutyCycleEncoder": //absolute encoder
+                dutyCycleEncoder = initializeDutyCycle(portNumber, positionOffset);
+                break;
             default:
-                System.err.println("Error: encoders not activated");
+                System.err.println("Error: single channel encoders not activated");
```

Comment:
```suggestion
```

Reply:
These start up error messages are intentional, and very helpful in determining that the robot is configured properly

Reply:
only prints once